css/styles.css

```diff
@@ -0,0 +1,680 @@
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  color: var(--black-color);
+  padding-bottom: 5em;
+}
+
+.tesxth {
+  padding-top: 4em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-size: x-large;
+  font-weight: 400;
+  color: var(--red-color);
+}
+
+.mainprogram {
+  background-color: var(--black-color);
+  padding-bottom: 2.5em;
+  width: 100%;
+  background-image: url(../img/00.png);
+}
+
+.mainprogram h2 {
+  text-align: center;
+  color: #fff;
+}
+
+.mainprogram hr {
+  width: 10%;
+  color: orange;
+  margin-bottom: 2em;
+}
+
+.toprow,
+.bottomrow {
+  display: flex;
+  justify-content: space-between;
+}
+
+.toprow img,
+.bottomrow img {
+  width: 12rem;
+  align-items: center;
+  display: flex;
+}
+
+.climate,
+.lehmann,
+.weleda,
+.rb,
+.elobau {
+  margin-top: 5%;
+}
+
+.farmimgpartnersabout {
+  background-color: var(--black-color);
+  padding-bottom: 2px;
+}
+
+.toppart h3 {
+  margin: auto;
+  padding: 0.2em;
+  color: var(--black-color);
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 400;
+  font-size: initial;
+  text-align: center;
+  padding-top: 2.5em;
+}
+
+.toppart hr {
+  display: block;
+  height: 0.2em;
+  background-color: #ec5242;
+  margin: auto;
+  width: 5em;
+  margin-bottom: 3em;
+}
+
+.farmimgpartners h3,
+.farmimgpartnersabout h3 {
+  padding-top: 1em;
+  padding-bottom: 0.1em;
+  align-items: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  color: var(--gray-color);
+  font-size: large;
+  margin-bottom: 0.5em;
+  text-align: center;
+}
+
+.farmimgpartners hr,
+.farmimgpartnersabout hr {
+  display: block;
+  height: 0.1em;
+  background-color: #ec5242;
+  margin: auto;
+  width: 3em;
+}
+
+.firstrow,
+.secondrow,
+.thirdrow {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2em;
+}
+
+.speaker {
+  display: inline-flex;
+  width: 49%;
+  margin-bottom: 3.5%;
+}
+
+.speakertext > h4 {
+  font-family: 'Cocogoose', sans-serif;
+  font-size: large;
+  font-weight: 400;
+  color: var(--black-color);
+  margin-bottom: 0;
+}
+
+.speakertext1 {
+  color: var(--red-color);
+  font-family: 'Lato', sans-serif;
+  font-style: italic;
+  font-size: smaller;
+  margin-top: 0;
+}
+
+.imagecontainer {
+  border: #d3d3d3 solid;
+  width: 75%;
+  margin: auto;
+  margin-bottom: 1.5em;
+}
+
+.home hr {
+  width: 10%;
+}
+
+.prevsumits {
+  padding-top: 5%;
+  background-color: #f7f2f2;
+}
+
+button:hover {
+  cursor: pointer;
+  background: #ec5242;
+  color: #000;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-ul {
+    margin-right: 8px;
+    flex-direction: row;
+  }
+
+  .hamburger {
+    display: none;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: white;
+  }
+
+  .nav-link a {
+    color: #d3d3d3;
+  }
+
+  .top-black-navbar {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    background-color: grey;
+    height: 2.5em;
+  }
```

Comment:
- [ ] Kindly make sure to use the same background color in order to look exactly as the provided design. :wink: 

   - ### YOUR DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203581176-b60ac20b-2575-4ac5-b43d-9baa411c838b.png)

   - ### PROVIDED DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203581250-0b9d45d3-50f9-4ebb-a5a9-97632d0b389e.png)